controls/V-72269.rb

```diff
@@ -76,27 +54,30 @@
 
 If a crontab file does not exist in the \"/etc/cron.daily\" that executes the
 \"ntpdate\" file, this is a finding."
-  tag "fix": "Edit the \"/etc/ntp.conf\" file and add or update an entry to define
-\"maxpoll\" to \"10\" as follows:
+  tag "fix": "Edit the \"/etc/ntp.conf\" file and add or update an entry to
+define \"maxpoll\" to \"10\" as follows:
 
 maxpoll 10
 
-If NTP was running and \"maxpoll\" was updated, the NTP service must be restarted:
+If NTP was running and \"maxpoll\" was updated, the NTP service must be
+restarted:
 
 # systemctl restart ntpd
 
 If NTP was not running, it must be started:
 
 # systemctl start ntpd"
+  tag "fix_id": "F-78623r3_fix"
 
   describe service('ntpd') do
     it { should be_running }
   end
   describe.one do
     describe command('grep maxpoll /etc/ntp.conf') do
```

Comment:
This should read the output of `ntpd --saveconfigquit=<somefilename>` instead of the config file since this is the actual config as `ntpd` sees it.